tests/phpunit-mysql8.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Use this file to start a MySQL8 database using Docker and then run the test suite on the MySQL8 database.
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd $DIR
+cd ..
+
+docker run --rm --name mysql8-magic-query-test -p 3306:3306 -p 33060:33060 -e MYSQL_ROOT_PASSWORD= -e MYSQL_ALLOW_EMPTY_PASSWORD=1 -d mysql:8 mysqld --default-authentication-plugin=mysql_native_password
+
+# Let's wait for MySQL 8 to start
+sleep 20
+
+vendor/bin/phpunit -c phpunit.xml.dist $NO_COVERAGE
```

Comment:
What is this `$NO_COVERAGE` variable ? Can you document it at the beginning of the file ?

Reply:
It's just a way to pass arguments to phpunit, `NO_COVERAGE=--no-coverage`, since it's a dev script, i don't think it's worth documenting

Reply:
It could just be a `$PHPUNIT_ARG` looking at the position? Or `$PHPUNIT_EXTRA_ARG`?